launcher/modplatform/flame/PackManifest.cpp

```diff
@@ -87,7 +82,16 @@ bool Flame::File::parseFromBytes(const QByteArray& bytes)
         // this is probably a mod, dunno what else could modpacks download
         targetFolder = "mods";
     }
-
+    if(!obj.contains("downloadUrl") || obj["downloadUrl"].isNull() || !obj["downloadUrl"].isString() || obj["downloadUrl"].toString().isEmpty()){
+        //either there somehow is an emtpy string as a link, or it's null either way it's invalid
+        //soft failing
+        return false;
+    }
+    QString rawUrl = Json::requireString(obj, "downloadUrl");
```

Comment:
Maybe you could move this up, change to `Json::ensureString` and only check if the string is empty in the `if`.